zeppelin-web/src/app/notebook/notebook-actionBar.html

```diff
@@ -15,7 +15,7 @@
   <h3>
     <div style="float: left; width: auto; max-width: 40%"
       ng-controller="ElasticInputCtrl as input">
-      <input type="text" pu-elastic-input class="form-control2" placeholder="{{noteName(note)}}" style="min-width: 0px; max-width: 95%;"
+      <input type="text" pu-elastic-input class="form-control2" placeholder="Note {{note.id}}" style="min-width: 0px; max-width: 95%;"
```

Comment:
@cuspymd Thanks for your comment! As you can see, I just wanted to fix the Korean input problem without any changes to the existing behavior. However, I'm not sure what value would be appropriate for this placeholder.